app/api/uploadFile/route.ts

```diff
@@ -1,5 +1,5 @@
 import { S3, PutObjectCommand, GetObjectCommand } from "@aws-sdk/client-s3";
```

Comment:
Use kebab-case for url like `upload-file`.

Reply:
resolved!